packages/database/src/client.ts

```diff
@@ -1,8 +1,9 @@
 import {PrismaClient} from '@prisma/client'
+import type {Decimal} from 'decimal.js'
```

Comment:
It doesn't seem like `Decimal` is directly exported as part of `Prisma` and since they are deprecating getting it off the `@prisma/client/runtime`, I decided it would be easiest to pull in the underlying `decimal.js` library and grab the type from that.